MapboxNavigation/NavigationMapView.swift

```diff
@@ -1525,3 +1607,51 @@ extension NavigationMapView {
         highlightedBuildingsLayer.fillExtrusionOpacity = NSExpression(format: "mgl_interpolate:withCurveType:parameters:stops:($zoomLevel, 'linear', nil, %@)", opacityStops)
     }
 }
+
+// MARK: - Delay Information
+
+extension NavigationMapView {
```

Comment:
Since `NavigationMapView` now contains several features we can expose them as separate extension files (e.g. for route lines, building highlighting, vanishing route line, delay information).